engine/install/ubuntu.md

```diff
@@ -167,7 +167,6 @@ from the repository.
    Engine and containerd, or go to the next step to install a specific version:
 
     ```bash
-    $ sudo apt-get update
```

Comment:
I don't think we can remove it here, because in the first steps above, we add the docker repository to the list of repositories; after adding, we need to run `apt-get update` to update the local package cache, so that `apt` knows which packages are available in that repository to install

Reply:
When adding using `add-apt-repository`, the `apt update` is automatically launched in the single command. 

In very old versions (we would have to check which version of ubuntu is supported in your documentation/by docker) this wasn't the case.